test/messaging/MessageBusReceiverUpgradeable.t.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.13;
+
+import "forge-std/Test.sol";
+import "../../contracts/messaging/MessageBusUpgradeable.sol";
+import "../../contracts/messaging/AuthVerifier.sol";
+
+import "@openzeppelin/contracts-4.5.0/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+contract MessageBusReceiverUpgradeableTest is Test {
+    MessageBusUpgradeable public messageBusReceiver;
+    AuthVerifier public authVerifier;
+
+    function setUp() public {
+        authVerifier = new AuthVerifier(address(1337));
+        MessageBusUpgradeable impl = new MessageBusUpgradeable();
+        // Setup proxy with needed logic and custom admin,
+        // we don't need to upgrade anything, so no need to setup ProxyAdmin
+        TransparentUpgradeableProxy proxy = new TransparentUpgradeableProxy(
+            address(impl),
+            address(420),
+            bytes("")
+        );
+        messageBusReceiver = MessageBusUpgradeable(address(proxy));
+        messageBusReceiver.initialize(address(0), address(authVerifier));
+    }
+
+    function addressToBytes32(address _addr) public pure returns (bytes32) {
+        return bytes32(uint256(uint160(_addr)));
+    }
+
+    function testAddressToBytes32() public {
```

Comment:
test not completed?

Reply:
@BlazeWasHere this is a copypasta from immutable contracts testing. I believe it's just a logging test. In any case, improved them a bit in [732250d](https://github.com/synapsecns/synapse-contracts/pull/149/commits/732250d00946b132e616fffc919657a4aa8a4053):
```js
[PASS] testAddressToBytes32() (gas: 4067)
Logs:
  bytes32: 0x000000000000000000000000de03e73c3785ce086ca85c6315df376a4a64c84b
  address: 0xde03e73c3785ce086ca85c6315df376a4a64c84b
[PASS] testAddressABIEncode() (gas: 9631)
Logs:
    abi.encode: 0x0000000000000000000000006f4e8eba4d337f874ab57478acc2cb5bacdc19c9
       bytes32: 0x0000000000000000000000006f4e8eba4d337f874ab57478acc2cb5bacdc19c9
  encodePacked: 0x6f4e8eba4d337f874ab57478acc2cb5bacdc19c9
       address: 0x6f4e8eba4d337f874ab57478acc2cb5bacdc19c9
```